java/src/jmri/managers/ProxyTurnoutManager.java

```diff
@@ -254,6 +255,38 @@ public String getEntryToolTip() {
         return "Enter a number from 1 to 9999"; // Basic number format help
     }
 
+    /** {@inheritDoc}
+     * @return outputInterval from first TurnoutManager
+     */
+    @Override
+    public int getOutputInterval() {
+        return ((TurnoutManager) getMgr(0)).getOutputInterval();
+    }
+
+    /**
+     * {@inheritDoc}
+     * set outputInterval on all known TurnoutManagers
+     */
+    @Override
+    public void setOutputInterval(int newInterval) {
+        log.debug("setOutputInterval called in ProxyTurnoutManager");
```

Comment:
It may not be a good idea to allow setting the interval values through the proxy manager.  Since the values are being stored in the individual managers, allowing the proxy manager to set all of them at once could cause connection specific timing to be lost

Reply:
This way of setting outputInterval fixed a hickup in ``jmri.jmrix.internal.InternalTurnoutManagerTest``, which showed to use the ProxyManager to reach the SystemConnectionMemo when setting a value via the InternalTurnoutManager.
I agree it should not be used in the application itself and have added that warning to the javadoc.

Reply:
Was that the test you modified in the InternalTurnoutManager test?

Reply:
Yes, so over there I added `` // only intended for testing, do not set interval via ProxyTurnoutManager in actual code``